modules/azurerm/sonar-base-instance/setup.tftpl

```diff
@@ -52,10 +58,12 @@ function install_azcli_from_internet() {
 }
 
 function install_deps() {
-    command -v unzip || install_yum_deps_from_internet unzip
-    command -v lvm2 || install_yum_deps_from_internet lvm2
-    command -v jq || install_yum_deps_from_internet https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm jq
-    command -v az || install_azcli_from_internet
+    install_yum_dep_from_internet unzip
+    install_yum_dep_from_internet lvm2
+    install_yum_dep_from_internet https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm epel-release
+    install_yum_dep_from_internet jq
+    install_yum_dep_from_internet wget
+    test -f /usr/local/bin/aws || install_awscli_from_internet
```

Comment:
you should test az rather than aws. where do you install az cli? Did you test this?

Reply:
I have now tested yes